demo/nuxt.config.ts

```diff
@@ -1,27 +1,25 @@
 export default defineNuxtConfig({
   modules: [
     // https://github.com/nuxt-modules/supabase
-    '@nuxtjs/supabase',
-    // UI lib (will be soon open sourced)
-    '@nuxthq/ui',
+    "@nuxtjs/supabase",
+     // https://github.com/nuxt/ui
+     "@nuxt/ui",
     // https://github.com/nuxt-modules/color-mode
-    '@nuxtjs/color-mode',
+    "@nuxtjs/color-mode",
```

Comment:
This can actually be removed since it's now automatically part of `@nuxt/ui` via https://ui.nuxt.com/getting-started/installation#modules

Reply:
Addressed